CountOnMe/Controller/Delegate+Extension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Delegate+Extension.swift
+//  CountOnMe
+//
+//  Created by Redouane on 08/03/2024.
+//  Copyright © 2024 Vincent Saluzzo. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// MARK: - Extension CalculatorManagerDelegate
+
+extension ViewController: CalculatorManagerDelegate {
+    func showError(_ error: CalculationError) {
```

Comment:
**question (non-blocking)**: Why limit this function to only `CalculatorManagerDelegate`?